build/connect.js

```diff
@@ -9,15 +9,15 @@ module.exports = function (grunt) {
     return {
         livereload: {
             options: {
-                hostname  : '0.0.0.0',
-                port      : 443,
+                hostname  : '127.0.0.1',
```

Comment:
had to change the post to run localhost since deriv-app, which uses the same port, has to be launched at the same time for debugging